packages/actors_api/src/actors/Verifier.ts

```diff
@@ -185,96 +168,32 @@ async function verifyPublicPresentation(
-  }
-  return false
-}
-
 /**
  * [ASYNC] Verifies the Claimer's presentation of [[Credential]]s.
  *
  * @param message The Claimer's presentation of the [[Credential]]s that should be verified, the result of [[createPresentation]].
  * @param session The Verifier's private verification session created in [[finalize]].
- * @param _latestAccumulators The list of the latest accumulators for each Attester which signed a [[Credential]] of this presentation.
- * @param _attesterPubKeys The privacy enhanced public keys of all [[AttesterIdentity]]s which signed the [[Credential]]s.
- * @throws When either latestAccumulators or attesterPubKeys are undefined.
  * @throws [[ERROR_MESSAGE_TYPE]].
  * @returns An object containing the keys
  * **verified** (which describes whether the [[Credential]]s could be verified)
  * and **claims** (an array of [[Claim]]s restricted on the disclosed attributes selected in [[requestPresentationForCtype]]).
  */
 export async function verifyPresentation(
   message: IMessage,
-  session: IVerifierSession,
-  _latestAccumulators?: gabi.Accumulator[],
-  _attesterPubKeys?: PublicAttesterIdentity[]
+  session: IVerifierSession
 ): Promise<{
   verified: boolean
   claims: Array<Partial<IRequestForAttestation | IAttestedClaim>>
 }> {
   // If we got a public presentation, check that the attestation is valid
-  if (message.body.type === MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES_CLASSIC) {
+  if (message.body.type === MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES) {
     const attestedClaims = message.body.content.map(
       AttestedClaim.fromAttestedClaim
     )
     return verifyPublicPresentation(attestedClaims, session)
   }
-
-  // if we got a privacy enhanced attestation, check that this was allowed by the verifier and
-  // verify the attestation
-  if (message.body.type === MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES_PE) {
-    const accFailure = !accumulatorArrTypeguard(_latestAccumulators)
-    const keyFailure = !publicAttesterArrTypeguard(_attesterPubKeys)
-    if (accFailure || keyFailure) {
-      throw ERROR_PE_VERIFICATION(accFailure, keyFailure)
-    }
-    if (session.allowedPrivacyEnhancement) {
-      const latestAccumulators = _latestAccumulators as gabi.Accumulator[]
-      const attesterPubKeys = (_attesterPubKeys as PublicAttesterIdentity[]).map(
-        (ai: PublicAttesterIdentity) => ai.publicGabiKey
-      )
-      const {
-        verified,
-        claims,
-      } = await gabi.Verifier.verifyCombinedPresentation({
-        proof: message.body.content,
-        verifierSession: session.privacyEnhancement,
-        latestAccumulators,
-        attesterPubKeys,
-      })
-      return { verified, claims }
-    }
-  } else {
-    throw ERROR_MESSAGE_TYPE(
-      message.body.type,
-      MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES_CLASSIC,
-      MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES_PE
-    )
-  }
-  return { verified: false, claims: [] }
+  throw SDKErrors.ERROR_MESSAGE_TYPE(
```

Comment:
Nitpick: you could negate the conditional and move the error up for better readability